pkg/logs/config/tests/complete/conf.d/integration.d/integration4.yaml

```diff
@@ -10,7 +10,7 @@ logs:
     service: nginx
     source: nginx
     sourcecategory: http_access
-    tags: env:prod, foo:bar
+    tags: env:prod,foo:bar
```

Comment:
Why exactly are you removing this space? This was put on purpose to ensure we supported spaces properly in config

Reply:
If I don't remove it, I get an error in the unit tests

Reply:
Do you still need this change here now that you figured out it was something to do with Viper?